nims-notes-things/src/components/notes.js

```diff
@@ -9,13 +10,17 @@ function Notes() {
 
   return (
     <div>
-      {notes.map((n) => {
+      {notes.map((note, index) => {
         return (
           <div>
```

Comment:
add a key here. When you are mapping over something, always add a unique key attribute... you can use `key={index}` here.

Reply:
meaning `<div key={index}>`